recipe/meta.yaml

```diff
@@ -18,11 +18,11 @@ requirements:
     build:
         - python  # [win]
         - cmake
-        - libtiff 4.0.6
+        - libtiff 4.0.*
```

Comment:
Cool. In which case, I don't think we need to worry about the patch version. If it does crop up though we can work around it in other ways (by providing a package which matches its name, version and build number exactly). Let me know if it is an issue though!